### ISSUE TITLE
Adopt a rsyslog exporter here?

### ISSUE THREAD

**Issue by filippog** [prometheus-community/community#43]
I'm using `rsyslog_exporter` (specifically @aleroyer's fork at https://github.com/aleroyer/rsyslog_exporter, thank you for your work!). I was originally using soundcloud's fork at https://github.com/soundcloud/rsyslog_exporter and consulting its network of repositories there seem to be quite the activity in terms of forks, which is great.

Though as an end-user I find it hard to follow multiple upstream repositories for a given exporter, would there be interest in adopting rsyslog_exporter in this repo and also link it from https://prometheus.io/docs/instrumenting/exporters/ ? In my opinion that would help users find and use a single exporter for rsyslog

thank you !

**Comment by SuperQ** [comment]
Sounds reasonable to me.

CC @matthiasr @aleroyer

**Comment by matthiasr** [comment]
Yes I like the idea. I made the changes that I needed for SoundCloud at the time to the rsyslog exporter, but I won't have capacity to maintain a version continuously. @aleroyer would you like to promote your fork into the prometheus-community org and maintain it there? It would also be cool, if you have the capacity, to integrate useful changes from other forks.

**Comment by aleroyer** [comment]
Hi everyone :wave: 
I see no issue so far to promote my fork here. I use the exporter in production in my company so it shouldn't be a problem to dedicate some time to it.
Do you have any guidelines or documentation I need to follow in order to maintain this repository in the prometheus-community org? I suppose there is a specific release process, maybe some libraries to integrate or update?

**Comment by matthiasr** [comment]
I am happy to sponsor this effort. @aleroyer I created a team and invited you to this organization. Please accept the invite, and initiate the transfer of the repository, whenever you are ready. I can then accept that transfer.

**Comment by aleroyer** [comment]
@matthiasr I accepted the invite and transfered the repository. It seems like you didn't had to accept that transfer as I can find it [here](https://github.com/prometheus-community/rsyslog_exporter).

What's next? :)

**Comment by matthiasr** [comment]
I think that's all :) I'll make a PR to add a MAINTAINERS.md to the project, and I closed all the outstanding ones on the SoundCloud fork.
